sstables/index_reader.hh

```diff
@@ -166,11 +166,13 @@ private:
     std::optional<deletion_time> _deletion_time;
 
     trust_promoted_index _trust_pi;
-    std::optional<column_values_fixed_lengths> _ck_values_fixed_lengths;
+    column_translation _ctr;
```

Comment:
Shouldn't you share it?

Reply:
column_translation is a wrapper for shared pointer.

Reply:
It's surprising, but that's how column_translation was designed.

Reply:
It's like Java.

Reply:
It's fine to have lightweight classes (like we have effective_replication_map_ptr, that, except for its name, is quite lightweight).

Reply:
I guess lack of _ptr at the end makes it surprising. And in-place update semantics of get_for_schema()